contrib/benitlux/.gitignore

```diff
@@ -1,3 +1,4 @@
 src/benitlux_serial.nit
 *.db
 *.email
+benitlux_corrections.txt
```

Comment:
As you wish, but I think that you are worrying too much.
With a more optimistic philosophy, there is no historical errors, only happy little accidents. :)